pep/app/utils/dom.ts

```diff
@@ -101,3 +102,37 @@ export async function loadXSLT(this: any): Promise<Document | null> {
         return null;
     }
 }
+
+/**
+ * Convert object to browser encoded string
+ *
+ * @export
+ * @param {*} object
+ * @return {*}
+ */
+export function convertToBrowserQueryParamString(object: any) {
```

Comment:
is there a reason we can't use our existing `serializeQueryParams()` util for this?

Reply:
Yes, they are formatted differently. He basically is expecting us to grab the window/browser query string and use that.

Reply:
i guess im just not seeing where the difference in formatting is, as that existing util does browser encoding and stuff as well, is it how it handles arrays or something?

if we need this second util though, it probably makes more sense to live in `utils/url.ts` than `dom.ts`